scripts/prow.py

```diff
@@ -0,0 +1,42 @@
+import pika, sys, os, json
+
+# Parse CLODUAMQP_URL (fallback to localhost)
+success = False
+url = os.environ.get('CLOUDAMQP_URL', 'amqps://fjuitskq:WJ9hHhwhVXsc6N7J7lLQrEVFeIiDdRxY@shrimp.rmq.cloudamqp.com/fjuitskq')
```

Comment:
Note: the default URL here is a temporary free public RabbitMQ server (with a ton of limitations), created for the purpose of POC. Actual implementation will  likely try to push this as a secret (prow multistage custom secret?)